packages/camera/camera_platform_interface/lib/src/types/media_settings.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: avoid_equals_and_hash_code_on_mutable_classes
+
+import 'resolution_preset.dart';
+
+/// recording media settings.
+class MediaSettings {
+  /// constructor
+  const MediaSettings({
+    this.resolutionPreset,
+    this.fps,
+    this.videoBitrate,
+    this.audioBitrate,
+    this.enableAudio = false,
+  });
+
+  /// resolution preset
+  final ResolutionPreset? resolutionPreset;
+
+  /// camera fps
```

Comment:
The docs for all these fields should start with a capital letter. Also, they could be a little more descriptive. I typically model my documentation after the docs of the native API. This field could say something like: 
`Rate at which frames should be captured by the camera in frames per second.` from https://developer.android.com/reference/android/media/MediaRecorder

Reply:
Done.